app/models/druid_version_zip.rb

```diff
@@ -147,12 +160,14 @@ def total_part_size
       .sum
   end
 
-  def moab_size
+  def moab_version_size
+    moab_version_files.map { |file_path| File.size(file_path) }.sum
```

Comment:
```suggestion
    moab_version_files.sum { |file_path| File.size(file_path) }
```

Reply:
good catch, and fixed on the prior method definition too